index.js

```diff
@@ -39,25 +44,40 @@ function has(target) {
   return hasProhibited;
 }
 
-function sanitize(target, options) {
+function sanitize(target, options, regex) {
```

Comment:
What would be the use case for accepting regex in an argument like this? Unless there is a very good reason, I'd prefer not to do this, since we would need to account for user-supplied input (which could be a malicious regex, etc).